### ISSUE TITLE
Cannot create an instance of ReactiveUI.Winforms.ReactiveUserControl`1[TViewModel] because Type.ContainsGenericParameters is true. 

### ISSUE THREAD

**Issue by tazu08408** [reactiveui/ReactiveUI#2664]
I got the error below.
　
　Cannot create an instance of  ReactiveUI.Winforms.ReactiveUserControl`1[TViewModel] because 
　Type.ContainsGenericParameters is true. 

I am using .NET5, WinForms. (on winddows10)
When I designed the screen, it was displayed normally, but after restarting Visual Studio, an error was displayed.

**Comment by glennawatson** [comment]
Seems like the designer dislikes the generic view class. 

Some people make their own base classes from ours to make it easier on the designers. 

Eg

```public class MyBaseView : ReactiveUserControl<MyViewModel> { }```

**Comment by glennawatson** [comment]
You'll then use that in your designer

**Comment by tazu08408** [comment]
@glennawatson 
I'm sorry, but I would like to know more about it.
Below is a code example.
In this case, how should I fix it?

```
[HomeView.cs]
    public partial class HomeView : ReactiveUserControl<HomeViewModel>
    {}

[HomeView.Designer.cs]
    partial class HomeView
    {
        private void InitializeComponent()
        { }
    }
```

**Comment by tazu08408** [comment]
@glennawatson 
I tried it but couldn't improve it.
![2021-02-03 14_49_45-WindowsFormsReactiveUI - Microsoft Visual Studio](https://user-images.githubusercontent.com/14774601/106704296-3a260b80-662f-11eb-8ddd-d480a7b57340.png)

**Comment by snalesso** [comment]
Did you use your `HomeView` as root control (instead of `UserControl` or `Window`) in the XAML?

**Comment by nlz242** [comment]
Using Winforms (so no XAML) like the OP, in .NET 5, i get the smae thing. As soon as i switch my UserControl to ReactiveUserControl<MyViewModel>, the editor can't display it anymore.
I've tried, as suggested by @glennawatson, to create a base class and inherit that base class in my actual view, but it didn't help.
Tried inheriting the base class into any (and both) partial classes (my view & the designer). No changes.

Any other ideas? In the meantime, i change the inheritance and i comment the WhenActivated block of my View when i want to modify it thru the editor... it's a bit clunky but it works...
